File: example/javalib/testing/5-test-stealing/foo/test/src/foo/RandomTestsE.java
package foo;

import org.junit.Test;

public class RandomTestsE extends RandomTestsUtils {
  @Test
  public void test1() throws Exception {
    testGreeting("Sage", 28);
  }

  @Test
  public void test2() throws Exception {
    testGreeting("Talia", 22);
  }

  @Test
  public void test3() throws Exception {
    testGreeting("Urban", 25);
  }

  @Test
  public void test4() throws Exception {
    testGreeting("Violet", 20);
  }
}

File: example/kotlinlib/testing/5-test-stealing/foo/test/src/foo/RandomTestsE.kt
package foo

import org.junit.jupiter.api.Test

class RandomTestsE : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Sage", 28)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Talia", 22)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Urban", 25)
    }

    @Test
    @Throws(Exception::class)
    fun test4() {
        testGreeting("Violet", 20)
    }
} 
